src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -194,13 +193,13 @@ class ServiceDetails extends React.Component<RouteComponentProps<ServiceId>, Ser
         <div className="page-header">
           <h2>
             Service{' '}
-            <Link to={kialiRoute('/services')} onClick={this.updateFilter}>
+            <Link to="/services" onClick={this.updateFilter}>
```

Comment:
Are we sure hard-coded links to routes are a good idea?

Reply:
This wasn’t introduced by this PR and the route is relative to the basepath (/console)
Remember that this PR is reverting the workaround introduced by other PR